Fig.SqlSettings/Fig.SqlSettings.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
Had to upgrade  to standard 2.1 because 2.0 didn't have DbProviderFactories class

Reply:
Not a huge sacrifice but excludes a few platforms. Maybe the DbProviderFactories was a bad idea. The nice thing with DbProviderFactories is that the sql module could pick up the connection from AppSettingsXml or AppSettingsJson, something like this:

```csharp
var settings = new SettingsBuilder()
    .UseAppSettingsJson()
    .UseSql(connectionStringName: "myConnectionString")
    .Build();
```

The alternative is to have the user create a connection and pass it along, or better a lambda:

```csharp
var settings = new SettingsBuilder()
    .UseAppSettingsXml()
    .UseSql(() => new SqlConnection(), myConnectionString)
    .Build();
```

So maybe your original solution was the best all along. Thoughts?
This also answers the question of how to pass the SqlLite connection, given that it implements `IDbConnection`

Reply:
The UseSql with AppSettings can be elegant with DbProviderFactory if you're willing to stick with standard 2.1,
Then we should probably match all the other projects with version 2.1 aswel?
But it seems like it does limit in memory databases, can always make some overload to provide your own IDbConnection
instead on the SqlSettingsSource for people who wanna read it from an in memory database.

Or we can keep it more basic with my previous solution (I can rework the db connection to be done in GetSettings instead of constructor)

Reply:
> Then we should probably match all the other projects with version 2.1 aswel?

.NET Standard 2.1 is a show stopper. It doesn't support .NET Framework!
https://docs.microsoft.com/en-us/dotnet/standard/net-standard

> Or we can keep it more basic with my previous solution (I can rework the db connection to be done in GetSettings instead of constructor)

Yes, I think this would be best so we can stay on 2.0

Reply:
Alright, i'll revert this commit and rework the constructor